### PR TITLE
[DOCFIX] Add test section to "Running Alluxio on Cluster with High Availability" doc

### PR DESCRIPTION
**Pull request by bf8086**

### CODE REVIEW

File: docs/_includes/Running-Alluxio-Fault-Tolerant/jps.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```bash
```
Comment (apc999) [inline docs/_includes/Running-Alluxio-Fault-Tolerant/jps.md]: @bf8086 this is the old convention but we are currently moving away from putting code snippets under `docs/_includes`.
Instead, simply inline the code in the original .md file. In this way, it is easier to edit and maintain.

File: docs/_includes/Running-Alluxio-Fault-Tolerant/kill-leader.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```bash
```
Comment (apc999) [inline docs/_includes/Running-Alluxio-Fault-Tolerant/kill-leader.md]: ditto

File: docs/_includes/Running-Alluxio-Fault-Tolerant/see-leader.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+```bash
```
Comment (apc999) [inline docs/_includes/Running-Alluxio-Fault-Tolerant/see-leader.md]: ditto

File: docs/_includes/Running-Alluxio-Fault-Tolerant/jps.md (hunk 1 above)
Comment (bf8086) [reply docs/_includes/Running-Alluxio-Fault-Tolerant/jps.md]: Done.

File: docs/en/Running-Alluxio-Fault-Tolerant.md [hunk 4]
```diff
@@ -152,3 +152,30 @@ Alluxio leader master.
 ```
 hadoop fs -ls alluxio:///directory
 ```
+
+### Automatic Fail Over
+
+To test automatic fail over, ssh into current Alluxio master leader, and find process ID of
+the AlluxioMaster process with:
```
Comment (gpang) [inline docs/en/Running-Alluxio-Fault-Tolerant.md]: NIT: use backticks around "AlluxioMaster" -> `AlluxioMaster`

Comment (bf8086) [reply docs/en/Running-Alluxio-Fault-Tolerant.md]: Done.

**Comment by bf8086** [comment]
I moved code snippets inline. @apc999 PTAL.

**Comment by bf8086** [comment]
Reopened as #6045 to target `branch-1.6`

**Pull request closed without merging**